docs/provisioner/09-01-provisioner-api-spec.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
As discussed, pls place this info on the spec somewhere in the overview doc as we are no longer using the `API` doc type.